### PR TITLE
#5864 Use built in linkify html type

### LINKED ISSUE #5864: Use built-in linkify-html types
Instead of https://github.com/FlowCrypt/flowcrypt-browser/blob/b8aa4bebd85f33d2fd15b3682db5da8d4a74002b/extension/types/linkifyHtml.d.ts let's use built-in `linkify-html` types.

### PR DESCRIPTION
**Pull request by ioanmo226**
This PR uses built in linkify html type

close #5864 // if this PR closes an issue

----------------------------------

**Tests** _(delete all except exactly one)_:
- Does not need tests (refactor only, docs or internal changes)

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [x] addresses the issue it closes (if any)
- [x] code is readable and understandable
- [x] is accompanied with tests, or tests are not needed
- [x] is documented clearly and usefully, or doesn't need documentation
- [x] is free of vulnerabilities

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
     "@types/chrome": "0.0.284",
     "@types/dompurify": "3.2.0",
     "@types/jquery": "3.5.32",
+    "@types/linkifyjs": "^2.1.7",
```
Comment (sosnovsky) [inline package.json:15]: here we don't need separate types package, as starting from v3 `linkifyjs` includes ts types in the main package - https://github.com/nfrasser/linkifyjs/releases/tag/v3.0.0, they're available at `node_modules/linkifyjs/dist/` or `node_modules/linkify-html/dist/`

Comment (ioanmo226) [reply package.json]: Yep. you are right!
Fixed

**Review by sosnovsky — DISMISSED** [review]
Works well 👍

**Review by sosnovsky — APPROVED** [review]
👍

**Pull request merged**